Data/Sys/GameSettings/RFP.ini

```diff
@@ -1,7 +1,13 @@
 # RFPE01, RFPJ01, RFPR01, RFPP01, RFPK01, RFPW01 - Wii Fit Plus and Wii Fit Plus Channel
 
+[Core]
+# Dualcore causes hangs in the Rhythm Parade mini game.
+CPUThread = False
+
 [Video_Hacks]
 # The Wii Fit Plus Channel shows Mii faces and a graph; however, EFBToTextureEnable must be disabled for this to work.
 # Since both the channel and the main game update these images, it must be disabled for both.  (They both have the same GameID, though.)
 # However, this has a performance impact, so if the Wii Fit Channel is not going to be used, this does not need to be disabled.
 #EFBToTextureEnable = False
```

Comment:
Seems like there's a space missing after '#'

Reply:
I believe that's intentional, as it's a commented-out line of code rather than a normal comment.